tutorials/03-astropy_dev_env.rst

```diff
@@ -0,0 +1,159 @@
+How to set up a development version of Astropy
+==============================================
+
+Found a bug? Know how to fix it? See a modification you need for your
+own project? This tutorial will teach you how to set up a conda
+environment for installing and modifying a developer version of Astropy.
+
+Pre-requisites
+--------------
+
+-  `Git <https://git-scm.com/>`__ version control system. On a Mac, this
+   will come with installing the latest version of the `Xcode
+   developer <https://developer.apple.com/xcode/>`__ tools.
+-  An account on `GitHub <https://github.com/>`__
+
+This tutorial will show you how to fork, clone, and install the
+development version of Astropy from the command line (terminal).
+
+Step 1: Fork the Astropy repo on Github
+---------------------------------------
+
+-  Log into your account on Github
+-  Go to https://github.com/astropy/astropy and click on **Fork** in the
```

Comment:
I'm not sure, would screenshots be an overkill here?

Reply:
Not overkill. More like a desired upgrade, which probably won't happen for awhile.